sympy/polys/domains/tests/test_domains.py

```diff
@@ -1273,3 +1273,65 @@ def test_AlgebraicField_alias():
     k = QQ.algebraic_field(alpha)
     beta = k.to_alg_num(k([1, 2, 3]))
     assert beta.alias is alpha.alias
+
+
+def test_exsqrt():
+    assert ZZ.is_square(ZZ(4))
```

Comment:
With a function that is supposed to return True/False it is better to assert with `is` in the test:
```python
assert ZZ.is_square(ZZ(4)) is True
```
Otherwise in Python most objects will test as true e.g.:
```python
assert object()
```